migrations/2024-12-15-151642_add_index_on_person_id_read_for_read_only_post_actions/up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX idx_post_actions_on_read_read_not_null ON post_actions (person_id, read)
```

Comment:
The added sort in post_view.rs results in incorrect behavior when a pagination cursor is used. I'm also now realizing that the sort for saved_only also has this problem. Later I will figure out a fix, which will be a little complicated because the cursor will need to use values from two tables. It doesn't need to be fixed in this PR, but the index should include the post_id tie breaker, which cursor pagination will need because some reads are inserted in bulk. Otherwise there would be two index creations instead of one because a later migration would need to replace this index.

```suggestion
CREATE INDEX idx_post_actions_on_read_read_not_null ON post_actions (person_id, read, post_id)
```

Reply:
okay I have added the changes and will now fix all woodpecker steps locally and then commit again.